sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -231,6 +231,42 @@ private[spark] object HiveUtils extends Logging {
     }.toMap
   }
 
+  private[hive] def newHiveConfigurations(
```

Comment:
can you add some comments for this method? Especially what's the difference between this one and `hiveClientConfigurations`.

Reply:
hiveClientConfigurations is used to change those time values in the form of "1s"/ "10min" to long values, we may give it a more proper name